### DISCUSSION TITLE
Spoken Chinese languages

### DISCUSSION THREAD

**Discussion by kruton** [openai/whisper#25] [Q&A] [answered]
The list of languages mentions Chinese which is imprecise when dealing with spoken languages which might share similar writing systems. Is it trained on spoken Mandarin, Cantonese, Hakka, something else, some of these, all of these, etc?

**Comment by jongwook** [comment, marked as answer]
When using `zh (Chinese)` as the language label, our `medium` and `large` models showed ~8% WER and a BLEU score of ~13 on the `yue_hant_hk` split of the Fleurs dataset, suggesting that it works quite decently on Cantonese and possibly on other dialects as well. We'd be interested in knowing how usable it is for the other Sinitic languages.

**Reply by ronyfadel** [reply]
How can you get transcriptions for Taiwan (traditional Chinese characters) v/s China (simplified Chinese characters)?

**Reply by glangford** [reply]
fyi 
* https://github.com/openai/whisper/discussions/366#discussioncomment-3920328

**Reply by miacias** [reply]
Cantonese is not a dialect; it is a distinct language from Mandarin. Both, among others, make up the 'Chinese' or Sinitic branch of the Sino-Tibetan language family. @kruton asked a valid question about whether the label marked as `zh Chinese` will make a distinction between these separate languages.

**Reply by ryanheise** [reply]
The large-v3 model was just announced (#1762 ) which introduces a separate language code for Cantonese. You can use `yue` or `Cantonese`.

Models prior to large-v3 (as mentioned above) are capable of transcribing both Mandarin and Cantonese (and possibly others), even though there was just a single `zh` label. Although with v3, the accuracy for Cantonese should be greater under the `yue` label than it was in v2 under the `zh` label.

**Reply by miacias** [reply]
Thank you so much for this info, this is exactly what I was hoping for!
